config_example.json

```
@@ -1223,15 +1223,16 @@
   },
   {
    "name": "CoinbasePro",
-   "enabled": true,
+   "enabled": false,
```

```suggestion
   "enabled": true,
```

Exchanges should be enabled by default